srcpkgs/consolation/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'consolation'
+pkgname=consolation
+version=0.0.8
+revision=1
+build_style=gnu-configure
+makedepends='pkg-config libinput-devel'
+short_desc='Libinput-based console pointer interface'
+maintainer='Nadeŭka <nadevko@riseup.net>'
+license=GPL-2.0-or-later
+homepage=https://salsa.debian.org/consolation-team/consolation
+distfiles=$DEBIAN_SITE/main/c/consolation/consolation_$version.orig.tar.xz
+checksum=65dce55d539bc5e8eb05bc1a097ffd64bb6901e915b059d477fa5eeec022a192
+nocross='Require cross compilation of pkg-config'
```

Comment:
This shouldn't be `nocross`, changing `pkg-config` to `hostmakedepends` should fix it.